### ISSUE TITLE
Use a meaningful link in the card component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#154]
### Problem

[molecules/molecules.card](https://a-ui.github.io/core_branding_scss/#molecules-card)

The link 'Read the article' (it's 'Read more' in the code example) doens't say which article this link points to. Because the link is in a separate paragraph, the purpose of the link cannot be 'programmatically determined'. The link text should be clear from the link text alone or from the sentence, paragraph or list item it is in. If this is not the case, it fails success criterium 2.4.4: Link Purpose (In Context). 

Screenreaders can summon a link list. With the current link text, it would be unclear where this link goes.

```html
<p class="u-text-right">
<a href="#" class="has-icon-right">Read more<span class="fa fa-angle-right"></span></a>
</p>
```

![card screenshot](https://user-images.githubusercontent.com/870049/98652658-43f3e780-233c-11eb-904b-20e2232955e5.jpg)

### Solution

1. Change the link to make it meaningful. For example, 'Read more about Aeneis'.
2. Or you can add an aria-labelledby attribute to the link that refers to both the id of the link and the id of the title of the card. The current link text will be overwritten by the elements with these id's. For example:

```html
<div class="m-card__body">
 <h4 id="card-heading" class="h5 u-margin-bottom-xs">Aeneis</h4>
 <p class="small u-text-bold u-margin-bottom-xs">May 4th 2017, 12:34</p>
 <p class="u-margin-bottom">...</p>
 <p class="u-text-right">
  <a id="card-link" aria-labelledby="card-link card-heading" href="#" class="has-icon-right">Read more<span class="fa fa-angle-right"></span></a>
 </p>
</div>
```
This will concatenate a id="card-link" and h4 id="card-heading" in that order as the new link text.

### Links and articles
* [Understanding Success Criterion 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context)
* [WCAG ARIA Technique: Using aria-labelledby for link purpose](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA7)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
